docs/docs/cmd/entra/organization/organization-list.mdx

```diff
@@ -30,6 +30,25 @@ All other properties return with null values. To read all properties, use at lea
 
 :::
```

Comment:
Let's move the info admonition to the `permissions` section. That way, we can also drop the `remarks` heading.